app/api/delivery_api/controllers/order_api.rb

```diff
@@ -0,0 +1,49 @@
+module DeliveryApi
+  module Controllers
+    class OrderApi < Root
+      helpers DeliveryApi::Helpers::CartHelpers
+      helpers do
+        params :order_params do
+          requires :order, type: Hash do
+            requires :first_name,     type: String, desc: 'Client name.'
+            requires :last_name,      type: String, desc: 'Client surname.'
+            requires :address,        type: String, desc: 'Delivery address.'
+            requires :user_number,    type: String, desc: 'Client number.'
+            requires :delivery_type,  type: String, desc: 'TakeOut | Delivery.'
+            requires :pay_type,       type: String, desc: 'Cash || Terminal.'
+            optional :promo_code,     type: String, desc: '7a8ccc0e35'
+            optional :description,    type: String, desc: 'Wishes.'
+          end
+        end
+      end
+
+      resources :orders do
+        before { authenticate! }
+
+        desc 'Make order', entity: DeliveryApi::Entities::OrderResponce
+        params { use :order_params }
+        post :create do
```

Comment:
no needs to specify `:create` here. According to REST, post to a plural name of resource means creating.

Let's change it to `post do`

Reply:
Yep, fixed :)

Reply:
> no needs to specify `:create` here. According to REST, post to a plural name of resource means creating.
> 
> Let's change it to `post do`

Yep, fixed :)